grabbit/src/main/groovy/com/twcable/grabbit/spring/batch/repository/JcrExecutionContextDao.groovy

```diff
@@ -306,4 +303,42 @@ class JcrExecutionContextDao extends AbstractJcrDao implements ExecutionContextD
         contextAsMap.each { key, value -> context.put(key, value) }
         context
     }
+
+    @Override
+    List<String> getJobExecutionContextPaths(List<String> jobExecutionResourcePaths) {
```

Comment:
I think it makes sense to add some exceptions try/catch here but not for all. For example, some variables like `jobExecutionId` below will never be NULL since the way its created .. guarantees for the `ID` to be always present. If its not there, then there are bigger issues.